modin/pandas/base.py

```diff
@@ -3127,6 +3127,56 @@ def __init__(
         ]
         self.__groups = self.__get_groups(*self.resample_args)
 
+    _attributes = [
+        "rule",
+        "axis",
+        "closed",
+        "label",
+        "convention",
+        "kind",
+        "loffset",
+        "base",
+        "on",
+        "level",
+        "origin",
+        "offset",
+    ]
+
+    def __getitem__(self, key):
+        if len(self._dataframe.columns.intersection(key)) != len(key):
+            bad_keys = list(set(key).difference(self._dataframe.columns))
+            raise KeyError(f"Columns not found: {str(bad_keys)[1:-1]}")
```

Comment:
```suggestion
            raise KeyError(f"Columns {str(missed_keys)[1:-1]} don't exist.")
```

What's need of conversion `bad_keys` to str and slicing?

Reply:
@Rubtsowa address this please

Reply:
This was changed

Reply:
Address question https://github.com/modin-project/modin/pull/3613#discussion_r741841662, please